src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__section">
+      <article class="stars stars--0">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+      </article>
+
+      <article class="stars stars--1">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading.